### PR TITLE
[ISSUE_TEMPLATE] Show Actual Link Buttom to ReactDocs/issues and stackoverflow

### PR DESCRIPTION
**Pull request by ryota-murakami**
## Summary
My secondary standing on the @necolas of Giants.(prev: playground react-native-web)

## Now 
Document and Question button jumping create issue page of `facebook/react`.
Unfortunately, there are some people who doesn't follow our ISSUE_TEMPLATE message.

=====================================================================

Screen Shot 2020-02-14 at 19 09 28

## After PR
Jumping directly to `https://github.com/reactjs/reactjs.org/issues/new` or `https://stackoverflow.com/questions/tagged/reactjs?tab=Newest` when user clicked Document or Question buttons.  

=====================================================================

Screen Shot 2020-02-14 at 19 10 22

## Test Plan
Putting exact same `.github/ISSUE_TEMPLATE/config.yml` file on my personal repository.
Your can see look and behavior following link.
https://github.com/ryota-murakami/react-garden/issues/new/choose

And this is official docs about `.github/ISSUE_TEMPLATE/config.yml` file.
https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

If you got something want to do in `.github/ISSUE_TEMPLATE/config.yml` from docs info, please feel free to modify that.

Thank you! 😀

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+contact_links:
+    - name: 📃 Documentation Issue
+      url: https://github.com/reactjs/reactjs.org/issues/new
+      about: This issue tracker is not for documentation issues. Please file documentation issues here.
+    - name: 🤔 Questions and Help
+      url: https://stackoverflow.com/questions/tagged/react
```
Comment (necolas) [inline .github/ISSUE_TEMPLATE/config.yml]: Let's change this URL to:

`https://reactjs.org/community/support.html`

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+      about: This issue tracker is not for questions. Please ask questions here.
```
Comment (necolas) [inline .github/ISSUE_TEMPLATE/config.yml]: Let's change this to:

This issue tracker is not for support questions. Please refer to the React community's help and discussion forums.

Comment (ryota-murakami) [reply .github/ISSUE_TEMPLATE/config.yml]: @necolas Thank you for review it, I reflected this!
https://github.com/facebook/react/pull/18039/commits/c1459ca6ec39fac2637b389ed279125723d34706

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (ryota-murakami) [reply .github/ISSUE_TEMPLATE/config.yml]: @necolas Reflected this too. c1459ca

**Comment by necolas** [comment]
Thank you!

**Pull request merged**